pkg/analyzer/scan.go

```diff
@@ -127,14 +150,40 @@ func parseDeployResource(podSpec v1.PodTemplateSpec, resourceCtx *common.Resourc
 			resourceCtx.Resource.Network = append(resourceCtx.Resource.Network, n)
 		}
 		for _, e := range container.Env {
-			if strings.Contains(e.Value, ":") {
-				resourceCtx.Resource.Envs = append(resourceCtx.Resource.Envs, e.Value)
+			value, isPotentialAddress := identifyAddressValue(e.Value)
+			if isPotentialAddress {
+				resourceCtx.Resource.Envs = append(resourceCtx.Resource.Envs, value)
+			}
+		}
+		for _, envFrom := range container.EnvFrom {
+			//TODO: add support for configMapKeyRef (https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/)
+			if envFrom.ConfigMapRef != nil {
+				resourceCtx.Resource.ConfigMapRef = envFrom.ConfigMapRef.Name
 			}
 		}
 	}
 	return nil
 }
 
+//identifyAddressValue checks if value is a potential service address (value is originated from deployment's env or configmap values)
+//It returns a string value (if it's a potential address it may be added with default port) and a bool inidcating
+//if this is indeed a data value of interest as a potential address
+//service addressess considered are of the form "[http://]<service name>:<port number>"
+func identifyAddressValue(value string) (string, bool) {
+	if strings.HasPrefix(value, "http://") && strings.Count(value, ":") == 1 {
+		//consider also cases such as "http://<service name>" with default http port
+		//TODO: could also be a case where value is address as a service name without port, since default port may be used
+		return value + ":80", true //add default port for http
+	}
+	if strings.Contains(value, ":") {
+		return value, true
+	}
+	//TODO: could be a service name as address without default port and without prefix of http://
+	//TODO: what about other protocols prefixes? (https?)
+	//TODO: consider only string values containing services names
```

Comment:
We should also support references to external websites, not just in-cluster services.
Maybe we can identify services by their typical dns entry `<service-name>.<namespace>.svc.cluster.local` (though just `<service-name>.<namespace>` or `<service-name>` are also legal).

Reply:
Should we open a separate issue for this? Also, external websites are currently not supported in the created links/connections.

Reply:
yes, we should probably open a new issue for that